go.mod

```diff
@@ -162,8 +162,10 @@ require (
 	k8s.io/kubelet v0.0.0
 	k8s.io/legacy-cloud-providers v0.0.0
 	k8s.io/metrics v0.0.0
+	k8s.io/node-api v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
I think these changes are caused by you using a different version of go from the officially-supported version, and you should just revert them before committing

Reply:
Ah thanks, will do